tests/test_crypt.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import mock
+import os
+import sys
+import unittest
+
+try:
+  reload
+except NameError:
+  # For Python3 (though importlib should be used, silly 3.3).
+  from imp import reload
+
+from oauth2client.client import HAS_OPENSSL
+from oauth2client.client import SignedJwtAssertionCredentials
+from oauth2client import crypt
+
+
+def datafile(filename):
+  f = open(os.path.join(os.path.dirname(__file__), 'data', filename), 'rb')
```

Comment:
pretty sure this whole function is just `pkgutil.get_data(__name__, os.path.join('data', filename))`. go ahead and give it a try before you decide it's unrelated. `;)`

not deeply important though, this code all needs some spring cleaning.